packages/paste-core/components/typography/src/Paragraph.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+
+interface Paragraph {
+  id?: never;
+  className?: never;
+  children: NonNullable<React.ReactNode>;
+}
+
+const handlePropValidation = ({children}: Paragraph): void => {
+  if (children == null) {
+    throw new Error(`[Paste: Typography Paragraph] Must have non-null children.`);
+  }
+};
+
+const Paragraph: React.FC<Paragraph> = props => {
+  handlePropValidation(props);
+
+  return (
+    <Text
```

Comment:
Should we explicitly set the `as` prop to set the element to be `p`, even though it's default? _Just in case?_

Or are we happy any change/regression would be caught by the snapshot test?

Reply:
Might as well be safe, and explicitly set it.